src/main/java/org/dbflute/intro/app/web/playsql/data/PlaysqlDataAction.java

```diff
@@ -0,0 +1,39 @@
+package org.dbflute.intro.app.web.playsql.data;
```

Comment:
Add copyright like a this.

```
/*
 * Copyright 2014-2020 the original author or authors.
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *     http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
 * either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */
```

Reply:
@EgumaYuto 
>Add copyright like a this.

Fixed.
c7fd4ce6eb6517117d5543fcaf69d38315858410